public/jira/rollup/percent-complete/percent-complete.test.ts

```diff
@@ -0,0 +1,813 @@
+              parentKey: "e-3",
+              key: "s-4",
+              derivedTiming: { totalDaysOfWork: null, completedDaysOfWork: 0 },
+            },
+            {
+              parentKey: "e-3",
+              key: "s-5",
+              derivedTiming: { totalDaysOfWork: null, completedDaysOfWork: 0 },
+            },
+          ],
+        ] as IssueOrRelease[][]
+      ).reverse();
+
+      const expected = [
+        {
+          rollupData: [
+            {
+              completedWorkingDays: 0,
+              remainingWorkingDays: 30,
+              totalWorkingDays: 30,
+              userSpecifiedValues: false,
+            },
+            {
+              completedWorkingDays: 0,
+              remainingWorkingDays: 30,
+              totalWorkingDays: 30,
+              userSpecifiedValues: false,
+            },
+          ],
+        },
+        {
+          rollupData: [
+            {
+              completedWorkingDays: 0,
+              remainingWorkingDays: 2000,
+              totalWorkingDays: 2000,
+              userSpecifiedValues: true,
+            },
+          ],
+        },
+        {
+          rollupData: [
+            {
+              completedWorkingDays: 0,
+              remainingWorkingDays: 2000,
+              totalWorkingDays: 2000,
+              userSpecifiedValues: true,
+            },
+          ],
+        },
+      ];
+
+      const actual = rollupPercentComplete(issuesAndReleases);
+
+      expect(actual).toHaveLength(expected.length);
+      for (const i of Array(actual.length).keys()) {
+        expect(actual[i].rollupData).toStrictEqual(expected[i].rollupData);
+      }
+    });
+    it("should use default estimate and completion if no estimates on stories, and fall back to epic estimates for epics and initiatives", () => {
+      const issuesAndReleases = (
+        [
+          [
+            {
+              parentKey: null,
+              key: "i-1",
+              derivedTiming: { totalDaysOfWork: 1, completedDaysOfWork: 0 },
+            },
+          ],
+          [
+            {
+              parentKey: "i-1",
+              key: "e-3",
+              derivedTiming: { totalDaysOfWork: 2000, completedDaysOfWork: 100 },
+            },
+          ],
+          [
+            {
+              parentKey: "e-3",
+              key: "s-4",
+              derivedTiming: { totalDaysOfWork: null, completedDaysOfWork: 10 },
+            },
+            {
+              parentKey: "e-3",
+              key: "s-5",
+              derivedTiming: { totalDaysOfWork: null, completedDaysOfWork: 20 },
+            },
+          ],
+        ] as IssueOrRelease[][]
+      ).reverse();
+
+      const expected = [
+        {
+          rollupData: [
+            {
+              completedWorkingDays: 0,
+              remainingWorkingDays: 30,
+              totalWorkingDays: 30,
+              userSpecifiedValues: false,
+            },
+            {
+              completedWorkingDays: 0,
```

Comment:
it's kinda odd that it doesn't use completed working days for the stories. It doesn't seem to match the name of the test.